util/packaging/common/build_helpers.sh

```diff
@@ -54,6 +55,14 @@ __build_packages() {
 
   pushd ${package_dir}
 
+  # Try to pull the latest version of this image, as a best effort.
+  docker pull $docker_image_base || true
+  # Whether we pulled successfully or not, use the image SHA256 digest to
+  # identify the copy of it we have locally, so build can proceed with an old
+  # version.
+  export DOCKER_IMAGE_NAME_FULL="$(docker inspect --format='{{index .RepoDigests 0}}' $docker_image_base)"
```

Comment:
nit, does this need to be exported or can it be `local`?

Reply:
For some reason in a previous iteration I thought it had to be exported. Changed to `local`